Tasks/AppCenterTest/task.json

```diff
@@ -11,7 +11,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 126,
+        "Minor": 129,
```

Comment:
Starting yesterday, the VSTS sprint is 130. This will work though.

Reply:
Sprints are 3 weeks long. M129 deployment starts today, so the sooner the better to merge into releases/m129.